overlord/assertstate/assertmgr.go

```diff
@@ -42,6 +42,8 @@ type AssertManager struct {
 
 // Manager returns a new assertion manager.
 func Manager(s *state.State) (*AssertManager, error) {
+	crossInit()
```

Comment:
Is it ok that this will be called everytime we create a manager? Or should we protect it via e.g. global `var once` and `once.Do(crossInit)` in Manager() ?

Reply:
I was wondering about the name, I think its fine but wonder if there migth be a better one. some ideas: lazyInit, globalInit, delayedInit. But as this is deep bikeshed land, feel free to ignore :)

Reply:
I thought about the first point, given that at real runtime the managers are essentially singletons created early before starting most other machinary I think it's ok, without the once.Do.

About the name I put "cross" in it because it's about things that concern across managers,  maybe delayedCrossMgrInit() would be slightly clearer?  (I don't want to make it super long either)

Reply:
ah, but we have AddCheckSnapCallback usages, for those now I'm using once.Do inside the delayedCrossMgrInit instead of the caller